build/PhiloGL.js

```diff
@@ -4504,6 +4504,223 @@ require('./modules/web.immediate');
 require('./modules/web.dom.iterable');
```

Comment:
I think we should gitignore the build directory and git rm this file. It's fine if the examples only run after npm installing and building.